server/bin/pbench-state-manager.py

```diff
@@ -69,21 +82,21 @@ def main(options):
                 return 2
             args["state"] = new_state
 
-        if "path" not in args and "name" not in args:
-            print(
-                f"{_NAME_}: Either --path or --name must be specified",
-                file=sys.stderr,
-            )
-            return 2
-
         # Either create a new dataset or attach to an existing dataset
         doit = Dataset.create if options.create else Dataset.attach
 
         # Find or create the specified dataset.
         doit(**args)
     except Exception as e:
         # Stringify any exception and report it; then fail
-        logger.exception("Failed")
+        logger.exception(
+            "Failed to {} {}", "create" if options.create else "attach", args
+        )
+        try:
+            dup = Dataset.query(resource_id=args["resource_id"])
+            logger.error("Found dup for {} : {}", args, dup.as_dict())
+        except Exception as d:
+            logger.error("Checking {}: {}", args["resource_id"], str(d))
```

Comment:
Do you want to ignore the dataset-not-found case, as that probably isn't an interesting error to report, here?  (That is, we should report a duplicate, and we should report access errors, but if the only problem is that that the dataset that we were trying to create is not here...well...um...duh?)

Reply:
This was more debugging code, which I don't think really serves a purpose. I just forgot to forget it.